001-core/password.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+add_action( 'user_profile_update_errors', 'validate_current_password', 1, 3 );
+
+function validate_current_password( $errors, $update, $user ) {
+	check_admin_referer( 'update-user_' . $user->ID );
+
+	if ( ! isset( $_POST['pass1'] ) || empty( $_POST['pass1'] ) || ! $update ) {
+		return;
+	}
+
+	if ( ! isset( $_POST['current_pass'] ) || empty( $_POST['current_pass'] ) ) {
+		$errors->add( 'wrong_current_password', __( '<strong>ERROR</strong>: Please enter your current password.' ), array( 'form-field' => 'current_pass' ) );
```

Comment:
Do we want to have a different error code for this condition? Right now it's the same as when the pw doesn't match, which could be confusing.

Reply:
For consistency with WP Core's error messages, can we use `Error`?
```suggestion
		$errors->add( 'wrong_current_password', __( '<strong>Error</strong>: Please enter your current password.' ), array( 'form-field' => 'current_pass' ) );
```
Here's a screenshot of a different profile form error:

<img width="358" alt="Screen Shot 2021-04-08 at 3 16 44 PM" src="https://user-images.githubusercontent.com/1103700/114097630-8aaf2480-987d-11eb-8aba-177fef82cdab.png">

Reply:
The text domain is missing on this and other i18n functions. This string isn't in the WordPress core files, so it would need a text domain to have it be loaded from elsewhere. But that hasn't yet been determined for this repo :-)

Reply:
Not sure what to do about the text domain if it hasn't been determined for this repo. Should we come up with one now or is it something we want to discuss further?

Reply:
Probably want to discuss further, so in the meantime, the i18n function is basically redundant.

Reply:
In that case, is it ok if I remove it or can we leave it there?

Reply:
There's going to be a few other strings that will need the i18n strings added, so I would say it can be removed.